sample/store-syslog.conf

```diff
@@ -51,9 +51,11 @@
   protocol http
   host 127.0.0.1
 
-  buffer_type file
-  buffer_path /var/spool/td-agent/buffer/groonga
-  flush_interval 1
+  <buffer>
+    @type file
+    buffer_path /var/spool/td-agent/buffer/groonga
```

Comment:
Yes, we can.
In buffer directive, path is correct parameter: https://github.com/fluent/fluentd/blob/master/lib/fluent/plugin/buf_file.rb#L36